utils/preDeployContracts.js

```diff
@@ -0,0 +1,53 @@
+const { deployContracts } = require('./deployContracts');
+
+const fs = require('fs-extra');
+
+module.exports = {
+  preDeployContracts: function(buildContractsPath, appPreBuiltContractArtifactsPath) {
+    fs.emptyDirSync(buildContractsPath, '');
+    process.env.TARGET_NETWORK = 'ropsten';
+    console.log('deploying to ropsten');
+    deployContracts()
+      .then(() => {
+        process.env.TARGET_NETWORK = 'kovan';
```

Comment:
Deploys to kovan and ropsten can be done in parallel in case this utility is time consuming.

Reply:
Good point, although I am not sure if that could raise an issue with writing to a single artifact file. The utility is not overly time consuming, considering that it will be run infrequently. Most of the time saved is in removing unwanted data from the artifact file.

Reply:
I went ahead and implemented the parallel deploys in https://github.com/magmo/devtools/pull/22/commits/8ddb102f335e230f9fd6e78f055c5c929ef2b9f3 . It's not as elegant as it could be, since the contracts end up being compiled multiple times (but this is reasonably fast). Fixing that would be out of scope.